frontend/package-lock.json

```diff
@@ -0,0 +1,14362 @@
+{
```

Comment:
This file is really huge, also it looks like it has some contents similar to frontend/package.json. Can they co-exist? Do the changes here need to go to package.json?

Reply:
I'm unsure as to what you mean can you clarify

Reply:
Sure. I actually meant two things. (1) I mean this file is really huge (14k lines), is there any way we can avoid it appear in a PR? (2) Then I looked at the repo and found package.json, whose name is similar. So I was asking should we put these changes into package.json?

Reply:
I beleive msot of the size of this file comes form pack-lock.json which is required as well as auto-generated by node. package-lock.json and package.json are different and they shoudlnt be combined so I was wondeirng if I should include it in another PR

Reply:
No worries. If it's only a one time cost, no need to split it, since it's just one auto-generated file. I was not sure what that file is.

Reply:
When we were setting up the React app, I believe [one of the docs said to delete the package-lock.json file from our directory](https://docs.google.com/document/d/1_DwUut7owgVkGYQXZO56Cj200wbUo5AiK917nOsYPlI/edit?usp=sharing).

Reply:
Ack